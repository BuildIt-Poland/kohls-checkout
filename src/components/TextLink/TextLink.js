import styled from 'styled-components';

import { colorTurquoise, fontWeightBold, fontMedium } from '../../styles/designTokens';

const TextLink = styled.a`
  color: ${colorTurquoise};
  font-weight: ${fontWeightBold};
  font-size: ${fontMedium};
  text-decoration: underline;
  text-transform: none;
`;

export default TextLink;
