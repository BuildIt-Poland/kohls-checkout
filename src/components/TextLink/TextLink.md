```js
<TextLink>Text Link</TextLink>
```
