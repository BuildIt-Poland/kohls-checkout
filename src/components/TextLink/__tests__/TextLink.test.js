import React from 'react';
import { create } from 'react-test-renderer';

import TextLink from '../TextLink';

describe('COMPONENT - TextLink', () => {
  it('renders correctly', () => {
    const component = create(<TextLink />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
