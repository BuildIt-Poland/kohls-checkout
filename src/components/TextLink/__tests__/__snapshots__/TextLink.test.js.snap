// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - TextLink renders correctly 1`] = `
.c0 {
  color: #00a09f;
  font-weight: 700;
  font-size: 1.8rem;
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

<a
  className="c0"
/>
`;
